apps/emqx/src/emqx_persistent_session_ds.erl

```diff
@@ -730,6 +712,15 @@ handle_info(
     Session#{s := S, shared_sub_s := SharedSubS};
 handle_info(AsyncReply = #poll_reply{}, Session, ClientInfo) ->
     push_now(handle_ds_reply(AsyncReply, Session, ClientInfo));
+handle_info(#new_stream_event{subref = Ref}, Session, _ClientInfo) ->
```

Comment:
We probably need a way to provide such an event from shared subs.

Reply:
Shared subscription coordinator can subscribe to the new stream events, then hand over streams to the sessions? I haven't touched that part, because I don't know it well enough.

Reply:
>Shared subscription coordinator can subscribe to the new stream events, then hand over streams to the sessions?

Yes, this is the most probable way of implementation. It may also keep periodical stream renewal for a while.

Nevermind, I recalled that sessions's stream renewal is only needed for proper streams.